### PR TITLE
add mongo connection code and db operation code example

### PR DESCRIPTION
**Pull request by Commodore68**
This will require an environment variable to be created for the backend

### CODE REVIEW

**Review by rcharl6 — APPROVED** [review]
Great job Chris, thanks!

**Review by BradKurzawa — APPROVED** [review]

**Review by sjohns98 — APPROVED** [review]

**Review by ellammiller — APPROVED** [review]

File: backend/src/database/mongo.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        const db = client.db();
+
+        result = CRUDFunction(db, params);
+
+        client.close((error) => {
+            console.log(`An error occurred closing the client: ${error}`);
+        });
+    });
+
+    return result;
```
Comment (Commodore68) [inline backend/src/database/mongo.ts]: add comments to this

File: backend/src/database/mongo.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+export const insertOneWrapper: MongoCRUDFunction = (db, otherArgs) => {
+    const {collection, data} = otherArgs;
+
+    db.collection(collection).insertOne(data, (error, res) => {
```
Comment (Commodore68) [inline backend/src/database/mongo.ts:53]: add comment for the expected shape of the data object

Comment (Commodore68) [reply backend/src/database/mongo.ts:53]: done in newer PR

File: backend/src/database/mongo.ts (hunk 1 above)
Comment (Commodore68) [reply backend/src/database/mongo.ts]: done in newer PR

**Pull request merged**